### PR TITLE
Added HeadSpin

### PR DESCRIPTION
**Pull request by jahmed2345**
Added HeadSpin

**Rendered article section for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/device-mode/testing-other-browsers?branch=pr-en-us-1748#cloud-based-emulators-and-simulators

todo: vchapel to move statement.

### CODE REVIEW

**Review by zoherghadyali — APPROVED** [review]

**Review by vchapel — APPROVED** [review]
LGTM

**Comment by jahmed2345** [comment]
Who will be able to merge this request?

**Review by vchapel — APPROVED** [review]
LGTM

**Review by zoherghadyali — APPROVED** [review]

File: microsoft-edge/devtools-guide-chromium/device-mode/testing-other-browsers.md [hunk 1]
```diff
@@ -120,18 +120,18 @@ If you aren't able to use the emulators and you don't have access to real device
 
 The following products are examples of cloud-based emulators and related products.  These descriptions are from the products' information sites and serve as examples of features or capabilities to consider.
 
-* [BrowserStack](https://www.browserstack.com/automate) helps you perform manual testing.  You select an operating system, a browser version, a device type, and a URL to browse, and then BrowserStack spins up a hosted virtual machine that you can interact with.  You can run multiple emulators in the same screen, to test the look and feel of your app across multiple devices at the same time.
+* [BrowserStack](https://www.browserstack.com/automate) helps you perform manual testing.  You select an operating system, a browser version, a device type, and a URL to browse, and then BrowserStack spins up a hosted virtual machine that you can interact with.  You can run multiple emulators in the same screen, to test the look and feel of your app across multiple devices at the same time. The following testing sites are suggested as a starting point. Conduct your own search to find the best cloud-based testing for your needs.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/device-mode/testing-other-browsers.md:123]:
```suggestion
* [BrowserStack](https://www.browserstack.com/automate) helps you perform manual testing.  You select an operating system, a browser version, a device type, and a URL to browse, and then BrowserStack spins up a hosted virtual machine that you can interact with.  You can run multiple emulators in the same screen, to test the look and feel of your app across multiple devices at the same time.
```

File: microsoft-edge/devtools-guide-chromium/device-mode/testing-other-browsers.md [hunk 2]
```diff
@@ -120,18 +120,18 @@ If you aren't able to use the emulators and you don't have access to real device
 
 The following products are examples of cloud-based emulators and related products.  These descriptions are from the products' information sites and serve as examples of features or capabilities to consider.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/device-mode/testing-other-browsers.md:121]:
```suggestion
The following products are examples of cloud-based emulators and related products.  These descriptions are from the products' information sites and serve as examples of features or capabilities to consider.  The following testing sites are suggested as a starting point. Conduct your own search to find the best cloud-based testing for your needs.
```
try to condense

**Review by mikehoffms — APPROVED** [review]

**Pull request merged**